features/waarden.feature

```diff
@@ -214,3 +339,99 @@ Feature: leveren van waarden bij een WOZ-object
         }
       ]
       """
+
+    Scenario: meerdere vernietigde beschikkingen met zelfde peildatum en ingangsdatum worden genegeerd
+      Gegeven /wozobjecten/051800027395 levert de volgende waarden:
+      """
+      [
+        {
+          "vastgesteldeWaarde": 210000,
+          "waardepeildatum": "2018-01-01",
+          "ingangsdatum": "2019-07-01",
+          "beschikkingsStatussen": [ "vernietigingbeschikking", "vernietigingbeschikking" ]
+        },{
+          "vastgesteldeWaarde": 295000,
+          "waardepeildatum": "2019-01-01",
+          "ingangsdatum": "2020-01-01",
+          "beschikkingsStatussen": [ "beschikking_genomen" ]
+        },{
+          "vastgesteldeWaarde": 283000,
+          "waardepeildatum": "2018-01-01",
+          "ingangsdatum": "2019-01-01",
+          "beschikkingsStatussen": [ "beschikking_genomen" ]
+        },{
+          "vastgesteldeWaarde": 274000,
+          "waardepeildatum": "2017-01-01",
+          "ingangsdatum": "2018-01-01",
+          "beschikkingsStatussen": [ "beschikking_genomen" ]
+        }
+      ]
+      """
+      Als de relevante waarden worden bepaald voor de geleverde waarden
+      Dan levert dit de waarden:
+      """
+      "waarden": [
+        {
+          "vastgesteldeWaarde": 295000,
+          "waardepeildatum": "2019-01-01"
+        },
+        {
+          "vastgesteldeWaarde": 283000,
+          "waardepeildatum": "2018-01-01"
+        },
+        {
+          "vastgesteldeWaarde": 274000,
+          "waardepeildatum": "2017-01-01"
+        }
+      ]
+      """
+
+    Scenario: indicatieBezwaarBeroep wanneer een van de statussen bezwaar, (hoger)beroep of cassatie ingesteld maar niet afgehandeld is.
```

Comment:
Ik denk dat het handiger is om hier een Abstract Scenario te gebruiken. Dan kan m.b.v. voorbeelden eenduidig worden aangegeven welke statussen leiden tot indicatieBezwaarBeroep

Reply:
heb ik nu gedaan, zie laatste versie